torchsynth/synth.py

```diff
@@ -1,7 +1,6 @@
 from typing import Any, Dict, List, Optional, Tuple
 
 import torch
-import torchcsprng as csprng
 from pytorch_lightning.core.lightning import LightningModule
 from torch import tensor as T
```

Comment:
import reproducibility? Or that's an option for abstract synth?

Reply:
See above -- it's in the package __init__